app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -291,4 +291,64 @@ public function validateBeforeOrderCommit($observer) {
         $orderModel = Mage::getModel('boltpay/order');
         $orderModel->validateBeforeOrderCommit($observer);
     }
+
+    /**
+     * Automatically create an invoice after creating the shipment if createInvoiceAfterCreatingShipment setting is set to yes
```

Comment:
@johnmccombs1   Can you please add the triggering event here like in all the other observer methods?